2016-06-28-Rust-Reach-Fractal.md

```diff
@@ -0,0 +1,91 @@
+# Where are we going with Tessel? Rust, Reach, and Fractal
+
+The aim of the Tessel Project is to create a fully open source hardware & software platform that makes it easy and intuitive to develop Internet-connected devices.
+
+With Tessel 1, we tackled the "easy and intuitive" part of that mission statement. Tessel 2 furthered that piece of the mission, and began to work towards letting developers build at scale. Specifically, I think we've done quite well towards fulfilling two of our core philosophies:
+
+> **Developer experience is paramount.** Tessel should be the fastest way to build an idea, regardless of your background (or lack thereof) in electrical engineering or computer programming. Branding, documentation, and engineering design decisions should always take this into account.
+
+> **Device design should focus on user experience rather than on implementation.** Tessel uses high level languages and modular plug & play hardware (see philosophy here) so that device creators can fully prototype and test device functionality before worrying about optimization and implementation specifics.
+
+Our next efforts should take us down the road to this less-fulfilled core belief:
+
+> **Tessel should be practical to use.** Tessel as a platform cannot have a big impact unless it is cost effective, reliable, and available for purchase. *It needs to be possible to purchase Tessel hardware at affordable prices, from quantity one up to production-level quantities (10,000+).*
```

Comment:
This should be an issue on MISSION.md in github.com/tessel/project - my text here is copy/paste from our core values